src/gp/inc/GPMSAOptions.h

```diff
@@ -119,6 +125,12 @@ class GPMSAOptions
   void set_autoscale_meanvar_scenario_parameter(unsigned int i);
 
   //! Do automatic normalization, using mean and variance of the
+  //  supplied data, for output i.
+  //
+  //  Normalized values will have mean 0 and standard deviation 1.
+  void set_autoscale_meanvar_output(unsigned int i);
```

Comment:
Yeah; I think when we add the partitioning capability, this won't be "output i" but rather "output subpartition i".